server/index.js

```diff
@@ -45,8 +69,3 @@ app.get('/tetraflix/dummyData/userMovies', (req, res) => {
   mgDummyData();
```

Comment:
Is this function synchronous? If not, you might want to make sure it returns a promise (or invokes res.send() as a callback) so that you don't send the response until the post request is processed and successful.

Reply:
The function is not synchronous, but I'm not sending anything back to the client from this function.  This adds a million users which takes about almost 2 hours.  I can't wait to send a response because there will be a timeout error.  I am only sending a string to the client to signal that the requests was received and users are currently being added.